src/bundle/RepositoryInstaller/Command/InstallPlatformCommand.php

```diff
@@ -282,7 +274,7 @@ private function executeCommand(OutputInterface $output, $cmd, $timeout = 300)
         );
 
         $process->run(static function ($type, $buffer) use ($output) { $output->write($buffer, false); });
-        if (!$process->getExitCode() === 1) {
+        if ($process->getExitCode() != self::SUCCESS) {
```

Comment:
how about a strict comparison?
```suggestion
        if ($process->getExitCode() !== self::SUCCESS) {
```